extendr-api/tests/extendr_macro.rs

```diff
@@ -130,6 +130,7 @@ fn tests_with_successful_outcomes() {
 // Win32 does not support catch_unwind.
 #[cfg(not(target_arch = "x86"))]
 #[test]
+#[ignore = "panicíng in FFI is now automatically abort instead of UB"]
```

Comment:
```suggestion
// This behavior is now handled in Rust nightly 1.81 making catch_r_error() unusable.
// For previous versions this is useful. 
// See related: https://rust-lang.zulipchat.com/#narrow/stream/122651-general/topic/.E2.9C.94.20.5BFFI.5D.20Is.20the.20representation.20of.20.26.5BT.3B.20N.5D.20and.20*const.20.5BT.3BN.5D
// TODO: revisit when formalized
// Win32 does not support catch_unwind.
#[cfg(not(target_arch = "x86"))]
#[test]
#[ignore = "panicking in FFI is now automatically abort instead of undefined behavior"]
```